### ISSUE TITLE
Sometimes causes errors/reboots when launching bubbles

### ISSUE THREAD

**Issue by AuroraWright** [GrapheneCt/LowMemMode#1]
With this plugin enabled, sometimes launching bubbles fails with an error code, or outright causes the Vita to reboot. A way I found to reproduce the issue is:
- Launch Adrenaline
- Launch settings without closing adrenaline
- Go to Themes, and pick the icon to download new themes from PSN
- When the PSN bubble is launched, the Vita reboots

Disabling the plugin fixes these issues.

**Comment by GrapheneCt** [comment]
Error code is C1-2609-7, right? The error is SCE_KERNEL_ERROR_CANNOT_ALLOC_PHYMEMLOW. It should only appear when you try to start some system apps with enlarged memory mode game and means that there is not enough RAM. If it appears in any other situations, can you describe it in more detail?

**Edit:** I can't reproduce reboot issue. Tried on two systems and PS Store launches fine on both of them.

What is your FW version? Can you provide list of all plugins that you are using?

**Comment by AuroraWright** [comment]
I just tried again and I can't reproduce it anymore, which is weird. I could have it happen predictably twice. I'm on 3.65 and my plugin list is as follows (plus yamt which is in the boot config)
```
*KERNEL
ur0:tai/vitabright.skprx
ur0:tai/NoPowerLimits.skprx
ur0:tai/music_premium.skprx
ur0:tai/PSVshell.skprx
ur0:tai/nonpdrm.skprx
ur0:tai/nopsmdrm.skprx
ur0:tai/repatch.skprx
ur0:tai/VideoDebug.skprx
ur0:tai/0syscall6.skprx
ur0:tai/itls.skprx
ur0:tai/udcd_uvc.skprx
ux0:app/PSPEMUCFW/sce_module/adrenaline_kernel.skprx

*main
ur0:tai/henkaku.suprx
ur0:tai/LowMemMode.suprx
ur0:tai/custom_warning_v2.suprx
ur0:tai/nolockscreen_v2.suprx
ur0:tai/shellbat.suprx
ur0:tai/jav.suprx
ur0:tai/quick_power.suprx
ur0:tai/quick_volume.suprx
ur0:tai/pngshot.suprx
ur0:tai/download_enabler.suprx

*NPXS10015
ur0:tai/henkaku.suprx

*NPXS10016
ur0:tai/henkaku.suprx

*NPXS10036
ur0:tai/reScreeny.suprx

*!PSPEMUCFW
*ALL
ur0:tai/PSMPatch.suprx
```
Though with the steps I described (plus opening a different bubble and having the OS close it before launching Adrenaline) I get error C1-2569-2 (not the one you mentioned) pretty consistently. One time, without closing anything after doing all that, I opened VitaShell and it kept having weird issues (errors on opening folders or on transferring files via FTP, not showing some partitions...) which I assume were due to lack of memory, and when I closed and reopened VitaShell it just failed outright with C1-2569-2.

**Comment by GrapheneCt** [comment]
Partially fixed in v1.1

**Comment by GrapheneCt** [comment]
Fixed in v1.2
